content/id/docs/tasks/administer-cluster/manage-resources/memory-constraint-namespace.md

```diff
@@ -0,0 +1,272 @@
+
+{{< codenew file="admin/resource/memory-constraints.yaml" >}}
+
+Membuat LimitRange:
+
+```shell
+kubectl apply -f https://k8s.io/examples/admin/resource/memory-constraints.yaml --namespace=constraints-mem-example
+```
+
+Melihat informasi mendetail mengenai LimitRange:
+
+```shell
+kubectl get limitrange mem-min-max-demo-lr --namespace=constraints-mem-example --output=yaml
+```
+
+Keluaran yang dihasilkan menunjukkan batasan minimum dan maksimum dari memori seperti yang diharapkan. Tetapi
+perhatikan hal berikut, meskipun kamu tidak menentukan nilai semula pada berkas konfirugasi untuk 
+LimitRange, nilai tersebut akan dibuat secara otomaris.
+
+```
+  limits:
+  - default:
+      memory: 1Gi
+    defaultRequest:
+      memory: 1Gi
+    max:
+      memory: 1Gi
+    min:
+      memory: 500Mi
+    type: Container
+```
+
+Mulai sekarang setiap Kontainer yang dibuat pada Namespace constraints-mem-example, Kubernetes
+akan menjalankan langkah-langkah berikut:
+
+* Jika Kontainer tersebut tidak menentukan permintaan dan limit memori, pasangkan nilai permintaan 
+dan limit memori pada Kontainer.
+
+* Memastikan Kontainer memiliki permintaan memori yang lebih besar atau sama dengan 500 MiB.
+
+* Memastikan Kontainer memiliki limit memori yang lebih kecil atau kurang dari 1 GiB.
+
+Berikut berkas konfigurasi Pod yang memiliki satu Kontainer. Manifes Kontainer
+menentukan permintaan memori 600 MiB dan limit memori 800 MiB. Nilai tersebut memenuhi
+batasan minimum dan maksimum memori yang ditentukan oleh LimitRange.
+
+{{< codenew file="admin/resource/memory-constraints-pod.yaml" >}}
+
+Membuat Pod:
+
+```shell
+kubectl apply -f https://k8s.io/examples/admin/resource/memory-constraints-pod.yaml --namespace=constraints-mem-example
+```
+
+Memastikan Kontainer pada Pod sudah berjalan:
+
+```shell
+kubectl get pod constraints-mem-demo --namespace=constraints-mem-example
+```
+
+Melihat informasi mendetail tentang Pod:
+
+```shell
+kubectl get pod constraints-mem-demo --output=yaml --namespace=constraints-mem-example
+```
+
+Keluaran yang dihasilkan menunjukkan Kontainer memiliki permintaan memori 600 MiB dan limit memori
+800 MiB. Nilai tersebut memenuhi batasan yang ditentukan oleh LimitRange.
+
+```yaml
+resources:
+  limits:
+     memory: 800Mi
+  requests:
+    memory: 600Mi
+```
+
+Menghapus Podmu:
+
+```shell
+kubectl delete pod constraints-mem-demo --namespace=constraints-mem-example
+```
+
+## Mencoba membuat Pod yang melebihi batasan maksimum memori
+
+Berikut berkas konfigurasi untuk sebuah Pod yang memiliki satu Kontainer. Kontainer tersebut menentukan
+permintaan memori 800 MiB dan batas memori 1.5 GiB.
+
+{{< codenew file="admin/resource/memory-constraints-pod-2.yaml" >}}
+
+Mencoba membuat Pod:
+
+```shell
+kubectl apply -f https://k8s.io/examples/admin/resource/memory-constraints-pod-2.yaml --namespace=constraints-mem-example
+```
+
+Keluaran yang dihasilkan menunjukkan Pod tidak dibuat, karena Kontainer menentukan limit memori yang
```

Comment:
Keep the API name "Container"
```suggestion
Keluaran yang dihasilkan menunjukkan Pod tidak dibuat, karena Container menentukan limit memori yang
```

Reply:
All resolved in e323b8c5920d6ae02c050306495e784b8e19ad9f